test/plugins/features/movies/controller.test.js

```diff
@@ -2,9 +2,55 @@
 
 const Controller = require('../../../../lib/plugins/features/movies/controller');
 const Movie      = require('../../../../lib/models/movie');
+const Knex       = require('../../../../lib/libraries/knex');
+
+const firstMovie  = { title: 'The Prestige', release_year: 2006 };
+const secondMovie = { title: 'Memento' };
+const thirdMovie  = { title: 'The Departed', release_year: 2006 };
 
 describe('movie controller', () => {
 
+  beforeEach(() => {
+    return Knex('movies').insert([firstMovie, secondMovie, thirdMovie]);
+  });
+
+  describe('filter', () => {
+
+    it('lists all movies', () => {
+      const filter = {};
+
+      return Controller.list(filter)
+      .then((movies) => {
+        expect(movies).to.have.length(3);
+        expect(movies.at(0).get('title')).to.eql(firstMovie.title);
+        expect(movies.at(1).get('title')).to.eql(secondMovie.title);
+        expect(movies.at(2).get('title')).to.eql(thirdMovie.title);
```

Comment:
Since you don't have any explicit ordering set in the controller, there's a slight chance that the movies won't always be returned in this order. Either way, your `expect` on line 24 is sufficient for the purpose of this test.